### ISSUE TITLE
TypeError: drag is null when scrolling in scheduler after changing state

### ISSUE THREAD

**Issue by lkrugertogetr** [telerik/kendo-react#1166]
## I'm submitting a...
* Bug report

## Current behavior
Scrolling in the scheduler after some actions (not all afaik) infinitely throws `TypeError: drag is null` (FireFox, in Chrome it throws `TypeError: Cannot read properties of null (reading 'element')`).

## Minimal reproduction of the problem with instructions
1. open https://www.telerik.com/kendo-react-ui/components/scheduler/
2. For your own sake, open the developer tools' debugger and enable pause on exception.
3. In the example, change the orientation to vertical.
4. Scroll in the scheduler.
5. Watch the errors come in.

## Environment

Package versions:

```
├── @progress/kendo-date-math@1.5.4
├── @progress/kendo-drawing@1.16.0
├── @progress/kendo-licensing@1.2.1
├── @progress/kendo-react-animation@4.13.0
├── @progress/kendo-react-buttons@4.13.0
├── @progress/kendo-react-dateinputs@4.13.0
├── @progress/kendo-react-dialogs@4.13.0
├── @progress/kendo-react-dropdowns@4.13.0
├── @progress/kendo-react-form@4.13.0
├── @progress/kendo-react-indicators@4.13.0
├── @progress/kendo-react-inputs@4.13.0
├── @progress/kendo-react-intl@4.13.0
├── @progress/kendo-react-labels@4.13.0
├── @progress/kendo-react-popup@4.13.0
├── @progress/kendo-react-scheduler@4.13.0
├── @progress/kendo-react-treeview@4.13.0
├── @progress/kendo-theme-default@4.43.0
├── @testing-library/jest-dom@5.16.1
├── @testing-library/react@12.1.2
├── @testing-library/user-event@13.5.0
├── @types/jest@27.0.3
├── @types/node@17.0.2
├── @types/react-dom@17.0.11
├── @types/react@17.0.37
├── @typescript-eslint/eslint-plugin@5.8.0
├── @typescript-eslint/parser@5.8.0
├── @typescript-eslint/typescript-estree@5.8.0
├── cldr-core@40.0.0
├── eslint-config-airbnb-typescript@16.1.0
├── eslint-config-airbnb@19.0.2
├── eslint-config-prettier@8.3.0
├── eslint-plugin-import@2.25.3
├── eslint-plugin-jsx-a11y@6.5.1
├── eslint-plugin-prettier@4.0.0
├── eslint-plugin-react-hooks@4.3.0
├── eslint-plugin-react@7.27.1
├── eslint@8.5.0
├── prettier@2.5.1
├── react-dom@17.0.2
├── react-scripts@5.0.0
├── react@17.0.2
├── sass@1.45.1
├── typescript@4.5.4
└── web-vitals@2.1.2
```

Browser:
- Chrome (desktop) 96
- Firefox version 95

System:
- TypeScript version: 4.5
- Node version: 16.x
- Platform: Windows

**Comment by kspeyanski** [comment]
Hello @lkrugertogetr 

Thank you for reporting this. We were able to reproduce it locally, but only with version `react@17` and `react-dom@17`.  

We will proceed further with the investigation and update the issue once we have something to share.

**Comment by Xizario** [comment]
Fixed in 4.14.0-dev.202201131241
https://stackblitz.com/edit/react-sbwxig?file=app/main.jsx
